tools/replication.go

```diff
@@ -0,0 +1,242 @@
+package tools
+
+// In /etc/hummingbird/andrewd-server.conf:
+// [replication]
```

Comment:
we should just hardcode all this right?

Reply:
I can if that's what you guys want. I'd rather we have the options but not advertise them, if that makes sense. Keep it in the code but not in the sample confs. As a quick example where some of these options would be useful in special cases: If a deployment is 100% SSD, you're probably going to want jobs_per_device to be greater than 1 since SSDs perform well with concurrency.